test/prevent-abbreviations.js

```diff
@@ -911,6 +926,17 @@ ruleTester.run('prevent-abbreviations', rule, {
 				}
 			`,
 			errors: createErrors()
+		},
+		// `checkFilenames` option
+		{
+			code: 'foo();',
+			filename: 'err/http-err.js',
+			errors: createErrors()
+		},
+		{
+			code: 'foo();',
+			filename: 'http-err.js',
```

Comment:
Could we also add a test for each kind of case from `getCase`? And also a test with an underscore (`_helper.js`) and dot (`.babelrc.js`) prefixes and a dot separator (`next.config.js`)

Reply:
will add more tests